x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/form/field_config.tsx

```diff
@@ -411,6 +411,28 @@ export const FIELD: Record<string, FieldMeta> = {
       };
     },
   },
+  [ConfigKey.ENABLED]: {
+    fieldKey: ConfigKey.ENABLED,
+    component: EuiSwitch,
+    label: i18n.translate('xpack.synthetics.monitorConfig.enabled.label', {
+      defaultMessage: 'Enable Monitor',
+    }),
+    controlled: true,
+    props: ({ isEdit, setValue }) => ({
+      id: 'syntheticsMontiorConfigIsEnabled',
+      label: isEdit
+        ? i18n.translate('xpack.synthetics.monitorConfig.edit.enabled.label', {
+            defaultMessage: 'Disabled monitors do not run tests.',
+          })
+        : i18n.translate('xpack.synthetics.monitorConfig.create.enabled.label', {
+            defaultMessage:
+              'Disabled monitors do not run tests. You can create a disabled monitor and enable it later',
```

Comment:
```suggestion
              'Disabled monitors do not run tests. You can create a disabled monitor and enable it later.',
```

cc @colleenmcginnis for content

Reply:
I'll go ahead and fix the punctuation. @colleenmcginnis LMK if you want any further change.

Reply:
Looks alright to me (though not totally sure where it's being used 🙃).

Reply:
@colleenmcginnis sorry about that, I don't have the code running to take a screenshot for you at the moment. If you click through to the original issue you can see [updated designs](https://github.com/elastic/kibana/issues/136687#issuecomment-1238536764) for the page that should include the copy, or at least the elements on the page should give you context.